UKHomeOffice/scm-repository-catalogue
# Replace AngularJS with React

## Context of this PR
We've replaced AngularJS with React using the NextJS framework and Typescript for ease of development and ongoing maintainability.

Also added the stacked bar charts for the apps installed within each organisation that we have data for.

## How to test?
- Check out the branch.
- Run npm install
- Run npm run dev - to check that you can run the local development server.
- Run npm run build-static - to compile a production version of the static site.

Caveats:
- Couple of loose ends that will need to be fixed in subsequent pull requests. e.g. Menu not collapse to a burger menu on mobile devices. Sorting of tables functionality will need to be re-added. Couple of lingering ng- references etc.
